remoting/server.go

```diff
@@ -243,3 +270,30 @@ func (s *server) closeNetConns() {
 		return true
 	})
 }
+
+// getServerTLSConfig builds the server tls config for remoting intra-cluster mTLS.
+func getServerTLSConfig(config conf.TLSConfig) (*tls.Config, error) {
```

Comment:
This looks very similar to the method in api/utils.go. Better to reuse that (move it to common)

Reply:
Done, extracted to `common` package